src/chipset/intel_82335.c

```diff
@@ -31,16 +31,25 @@
 #include <86box/port_92.h>
 #include <86box/chipset.h>
 
+/* Shadow capabilities */
 #define disabled_shadow (MEM_READ_EXTANY | MEM_WRITE_EXTANY)
 #define rw_shadow (MEM_READ_INTERNAL | MEM_WRITE_INTERNAL)
 #define ro_shadow (MEM_READ_INTERNAL | MEM_WRITE_DISABLED)
 
-#define extended_granuality_enabled (dev->regs[0x2c] & 0x01)
+/* Granularity Register Enable & Recalc */
+#define extended_granularity_enabled (dev->regs[0x2c] & 0x01)
+#define granularity_recalc ((dev->regs[0x2e] & (1 << (i+8))) ? ((dev->regs[0x2e] & (1 << i)) ? ro_shadow : rw_shadow) : disabled_shadow)
+
+/* R/W operator for the Video RAM region */
 #define determine_video_ram_write_access ((dev->regs[0x22] & (0x08 << 8)) ? rw_shadow : ro_shadow)
 
+/* Base System 512/640KB switch */
 #define enable_top_128kb (MEM_READ_INTERNAL | MEM_WRITE_INTERNAL)
 #define disable_top_128kb (MEM_READ_DISABLED | MEM_WRITE_DISABLED)
 
+/* ROM size determination */
+#define rom_size ((dev->regs[0x22] & (0x01 << 8)) ? 0xe0000 : 0xf0000)
+
```

Comment:
It's common practice to name #define's using ALL CAPS.

Reply:
I did it mostly in sake of clarity. Even if they do the same function. It's the first time I use that strategy.

Also all defines have been capped now on the new commit below